src/assets/scripts/client/aircraft/Pilot/Pilot.js

```diff
@@ -222,16 +240,22 @@ export default class Pilot {
      * @param {Number} speed - the speed to maintain, in knots
      * @return {Array} [success of operation, readback]
      */
-    maintainSpeed(currentSpeed, speed) {
+    maintainSpeed(currentSpeed, speed, aircraftModel) {
         const instruction = radio_trend('speed', currentSpeed, speed);
+        let speedUnattainable = '';
 
         this._mcp.setSpeedHold();
         this._mcp.setSpeedFieldValue(speed);
 
         // Build the readback
         const readback = {};
-        readback.log = `${instruction} ${speed}`;
-        readback.say = `${instruction} ${radio_spellOut(speed)}`;
+        readback.log = `${speedUnattainable}${instruction} ${speed}`;
+        readback.say = `${speedUnattainable}${instruction} ${radio_spellOut(speed)}`;
+        
+        if (speed > aircraftModel.model.speed.max || speed < aircraftModel.model.speed.min) {
+            speedUnattainable = 'requested speed unattainable, ';
```

Comment:
does this assignment actually have an effect?  my guess is no.  could you add at least one test for this case?